src/Drivers/Azki/Azki.php

```diff
@@ -175,7 +179,16 @@ private function getItems()
          *
          */
 
-        return $this->invoice->getDetails()['items'];
+        $new_items = array_map(
+            function ($item) {
+                $item['amount'] *= 10;  // convert toman to rial
```

Comment:
in this package, everything should be in toman! so this line is not valid.

Reply:
@khanzadimahdi 
Azki Gateway accept total amount price and items price in "Rial " unit.

as you can see in the previous PR #159  i also multiply the amount by 10 but i forgot to add it for items.
https://github.com/shetabit/multipay/blob/a063d5237bfc904149e4b0328d5c9de13ca9f03a/src/Drivers/Azki/Azki.php#L94-L103

also see ZarinPal, Sepehr, Walleta, etc driver  multiply by 10 
https://github.com/shetabit/multipay/blob/a063d5237bfc904149e4b0328d5c9de13ca9f03a/src/Drivers/Zarinpal/Strategies/Normal.php#L78-L84

this PR will fix this issue.